user/utils.py

```diff
@@ -0,0 +1,29 @@
+import jwt
+import json
+
+from django.http    import JsonResponse
+
+from my_settings    import SECRET_KEY, ALGORITHM
+from user.models    import User
+
+email_regax    = '^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$'
+password_regax = '[A-Za-z0-9@#$]{10,1000}'
+
+
+def login_decorator(func):
+    def wrapper(self,request, *args, **kwargs):
+        try:
+            token        = request.headers.get("Authorization")
+            payload      = jwt.decode(token, SECRET_KEY, ALGORITHM)
+            user         = User.objects.get(id=payload['use-id'])
+            request.user = user
+
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({'MESSAGE' : 'INVALID_USER'})
+        
+        except User.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_USER'})
+
+        return func(self, request, *args, **kwargs)
```

Comment:
return문을 try문 안으로 넣어주세요~
```suggestion
        return func(self, request, *args, **kwargs)
```

Reply:
앗 네 수정했습니다 !